warehouse/admin/views/sponsors.py

```diff
@@ -95,6 +101,22 @@ def sponsor_list(request):
     return {"sponsors": sponsors}
 
 
+def _handle_images(request, form):
+    if request.POST.get("white_logo", b"") != b"":
+        with tempfile.NamedTemporaryFile() as fp:
+            fp.write(request.POST["white_logo"].file.read())
+            storage = request.find_service(ISponsorLogoStorage)
+            extension = os.path.splitext(request.POST["white_logo"].filename)[-1]
+            filename = f"{sponsor.id}-white-logo{extension}"
+            form.white_logo_url.data = storage.store(filename, fp.name)
+    if request.POST.get("color_logo", b"") != b"":
+        with tempfile.NamedTemporaryFile() as fp:
+            storage = request.find_service(ISponsorLogoStorage)
+            fp.write(request.POST["color_logo"].file.read())
+            extension = os.path.splitext(request.POST["white_logo"].filename)[-1]
+            filename = f"{sponsor.id}-color-logo{extension}"
+            form.color_logo_url.data = storage.store(filename, fp.name)
```

Comment:
This is great! By updating `color/white_logo_url` we won't have to change anything were the logo is being displayed :+1: